es-ES improvement

jerosajose opened the pull request:
Improved version of the one requested by "Alvariithoo".

Alvariithoo commented:
> 2e76a5c not accurate at all, there are changes that confuse what it means other seems to be translated and not says something different than they really are 

> 8261d99 is OK

jerosajose commented:
It's not meant to confuse the player. The wording of many tips and hack names were changed to be more understandable at a first sight and be better expressed in Spanish, avoiding vague structures and making things clearer in less words.

I don't understand what you mean by "were translated", that's the point of this repository.

IMO, it's all the accurate it has to be.

absoIute commented:
Some of the changes here might be too long, (check HACK_NO_DO_NOT_FLIP, HACK_COPY_HACK), hack names need to be short to fit in the buttons

Alvariithoo commented:
Translating doesnt work and it shouldnt be that way, I learned for a long time that translating isnt always correct, what you do is interpret the language to another language is the correct way to do it

I didn't ignore you, sorry for that, it was very late at night

Now with the topic, there are some things that are better without changing, I'll just mention the lines in 2e76a5c:
Lines: 48, 87, 106, 111, 128, 129, 159, 251

Thanks for correct some part, so its all looks good to me

jerosajose commented:
> Translating doesnt work and it shouldnt be that way, I learned for a long time that translating isnt always correct, what you do is interpret the language to another language is the correct way to do it

Again, I don't understand what you mean with "translating", as you've indeed translated some terms that are better understood in English like "Noclip".

jerosajose commented:
Here's my view on the lines you mentioned:

- Line 48: I don't see any problem with the changes I made. The name is right, as "inviertido" is wrongly spelled and my name for the hack is more reduced in length. The tip is a bit more formal redacted and descriptive.

- Line 87: Again, the name wasn't descriptive enough so I went a bit further.

- Line 106: The word "fluida" is more accurate than "suave". Besides that, it was just a mere re-redaction to make it a bit more formally expressed, as the description of increasing the refresh rate of the animation on screen seemed a bit vague to me. If you mention "cada fotograma", you have to say respect to what, it won't be generating infinite FPS until it tops the GPU. You can limit yourself saying "Aumenta la frecuencia" and don't get too much into the matter. "La línea de playtest" seemed too informal or poorly written to me, so I replaced it with "cursor", which is correct as it is a line that moves following a rhythm/music/pace and is also widely refered as that in non-lineal video editing.

- Line 111: Once more, I rewrote the main tag to make it more descriptive at first sight.

- Line 128-129: "Limite" is a wrong interpretation of a threshold and is not really justified in any case. The right word for that is "umbral". Refer to the Spanish dictionary of the public institution RAE.

- Line 159: You made a typo there or any mistake copy/pasting the lines, as you mentioned practice mode in the hack for the normal mode confirmation message. I don't really understand how you can debate that, acknowledge that it is a mistake.

- Line 251: A more descriptive name for first sight read, nothing too special about that change. "Cofres rápidos" may suggest a new type of chests, and it's not the case. Note that this interface has to be friendly to all new players and customers of Mega Hack.

jerosajose commented:
> Some of the changes here might be too long, (check HACK_NO_DO_NOT_FLIP, HACK_COPY_HACK), hack names need to be short to fit in the buttons

Done, I've shortened the names of most hacks to the minimum possible in the last commit.

Alvariithoo commented:
People don't want to use a dictionary to understand, as I said before translated just not work, this need be simplified for the user not for us, we need think in the user who just want to play and understand what is using and reading

Im not trying to say you are wrong, Is not hate or something with you I hope you understand

jerosajose commented:
There's no strange words and anybody is able to understand what's written in there. If you don't really understand any word (which I don't think so if you're a native Spanish speaker), you should consult the dictionary. That's what kids with little vocabulary do.

I advised you earlier to refer to the dictionary because "límite" isn't the word for "threshold", no matter what kind of simplified view you want to give to the meaning of that tool tag. The best simplification comes to using the least words while expressing the same or more, and that's what I tried to achieve here. Making use of wrong words and an excessively reduced vocabulary isn't simplifying, it is to worsen the quality of the texts.

I also recommend you to improve your English before heading into this kind of works.
